Makefile

```diff
@@ -20,10 +18,7 @@ ifeq "$(ELPIDIR)" ""
 ELPIDIR=$(shell which elpi >/dev/null 2>&1 && elpi -where)
 endif
 ifeq "$(ELPIDIR)" ""
-ELPIDIR=elpi/findlib/elpi
-$(info Using elpi from the git submodule ./elpi/, override via ELPIDIR or PATH)
-else
-$(info Using elpi found in $(ELPIDIR), from ELPIDIR or PATH)
+$(error Elpi not found, make sure it is installed)
```

Comment:
`make sure elpi is in your path or set ELPIDIR`

Reply:
Changed.

Reply:
1 minute too late :D